compliance/nvidia/TEST04/README.md

```diff
@@ -0,0 +1,65 @@
+# Test 04 : Verify SUT is not caching samples
+## Introduction
+
+The purpose of this test is to ensure that results are not cached on the fly when SUT sees duplicate sample IDs.
+
+This test requires measuring & comparing performance of SUT in standard (PerformanceOnly, mode=2) mode versus
+the following mode:
+
+- TEST04 - Issue same sample: 
+	- Offline scenario, the same sample is repeated as many times as necessary to fill the query, targeting a minimum runtime duration of
+                                              at least 10 minutes. This breaks the requirement
+                                              of reading contiguous memory locations in Offline mode, but it is normal for an audit test, meant to 
+                                              stress the SUT in newer ways, to cause performance degradation.
+	- Single-Stream/MultiStream/Server scenario test ends after sending the same query as many times as necessary to satisfy a minimum duration of 10 minutes.
+
+## Prerequisites
+Test script works best with Python 3.3 or later.
+
+## Exempt Benchmarks
+This test is not applicable for the following benchmarks whose performance is dependent on variably sized input samples:
+ 1. rnnt
+ 2. bert
+ 3. dlrm
+ 4. 3d-unet
+
+## Scenarios
+
+ - This test is applicable for all scenarios.
+
+## Pass Criteria
+Performance of TEST04 should not be faster than the standard performance run in a statistically significant way. To account for noise, TEST04 can be at most 10% faster than the standard performance run.
+
+## Instructions
+
+### Part II : Run TEST04
+ - Copy provided [audit.config](https://github.com/mlperf/inference/blob/master/v0.7/compliance/nvidia/TEST04/audit.config) file in TEST04 folder to the corresponding benchmark directory from where the test is run
+ - Run the benchmark
+ - Verification that audit.config was properly read can be done by checking that loadgen has found audit.config in `mlperf_log_detail.txt`
+ - `mlperf_log_detail.txt` and `mlperf_log_summary.txt` files from this run are required to be submitted under TEST04-B
```

Comment:
TEST04-B?

Reply:
Thanks for your feedback, Anton. Much appreciated. Will address this.